lib/models.js

```diff
@@ -76,6 +80,13 @@ class DataElementSpecifications {
     return [];
   }
 
+  abstractsByNamespace(namespace) {
+    if (this._nsMap.has(namespace)) {
+      return this.byNamespace(namespace).filter(de => de.isAbstract);
+    }
+    return [];
+  }
```

Comment:
Same as `abstracts` above.  Is there a use case for when we would need to retrieve only the abstracts from a namespace?